FastSimulation/Tracking/python/SeedingMigration.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+
+def _regionProducerToFactoryPSet(producer):
```

Comment:
Let's do the simple parts here.
Tracking fastsim support is a part of TRK POG.
If the impact of this migration wasn't discussed yet with them, it should